### PR TITLE
Rookout driver docs

### PR DESCRIPTION
**Pull request by lirook**
I have added documentation for the Rookout driver.
The driver can be found at https://github.com/Rookout/rookout-nomad-driver

### CODE REVIEW

File: website/content/docs/drivers/external/rookout.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+1. Clone the driver's repo: `git clone https://github.com/rookout/rookout-nomad-driver`
+2. Change to the cloned dir: `cd rookout-nomad-driver`
+3. Create a plugin dir (unless you already have one for your Nomad installation): `mkdir plugins`
+4. Copy the corresponding binary from the `bin` folder, for example: `cp ./bin/java-rookout-linux-x64 plugins/java-rookout`
+5. Make sure that the driver has execution permission: `chmod +x ./plugins/java-rookout`
+6. When running the Nomad agent, be sure to set the plugins dir accordingly, for example: `nomad agent -dev -plugin-dir=/home/user/rookout-nomad-driver/plugins`
+
+## Task Configuration
+
+The `java-rookout` driver accepts all configuration options of the `java` driver.
```
Comment (jrasell) [inline website/content/docs/drivers/external/rookout.mdx:32]: Could we link out to the Java driver configuration options here?

**Review by jrasell — CHANGES_REQUESTED** [review]
Hi @lirook and thanks very much for this PR and the driver work. Would you be able to update the [index](https://github.com/hashicorp/nomad/blob/main/website/content/docs/drivers/external/index.mdx) page to include a link to this new page?

File: website/content/docs/drivers/external/rookout.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Rookout Driver
+
+Name: `java-rookout`
+
+[Rookout](https://rookout.com) is a disruptive developer solution for Cloud Native debugging and live-data collection. Rookout’s Non-Breaking Breakpoints let you collect any type of data on the fly with no extra coding, redeployments, or restarts.
+
+The rookout Nomad driver is used to execute Java applications packaged into a Java Jar file and automatically add the Rookout SDK to them.
+
+More info about the Rookout SDK is available in the [Rookout docs](https://docs.rookout.com).
```
Comment (jrasell) [inline website/content/docs/drivers/external/rookout.mdx:15]:
```suggestion
The Rookout Nomad driver is used to execute Java applications packaged into a Java Jar file and automatically add the Rookout SDK to them.

[Rookout](https://rookout.com) is a developer solution for Cloud Native debugging and live-data collection. Rookout’s Non-Breaking Breakpoints let you collect any type of data on the fly with no extra coding, redeployments, or restarts.

More info about the Rookout SDK is available in the [Rookout docs](https://docs.rookout.com).
```

**Review by jrasell — CHANGES_REQUESTED** [review]
Thanks for the addition commits @lirook. I have one final change suggestion which alters the ordering of the description and removes the word "disruptive". This is a follow up due to some async internal discussion.

I will also push a commit before merging to move the ordering on the index page to alphabetical; so there is no bias or ambiguity on where additions should be placed.

**Comment by lirook** [comment]
Thank you @jrasell !
I committed the suggested changes.
Let me know if there's anything else you need me to update.

**Review by jrasell — APPROVED** [review]
LGTM thanks @lirook!

**Pull request merged**